components/Navbar/Navbar.module.css

```diff
@@ -14,29 +14,25 @@
 
 .navitem {
   margin-left: 5rem;
+  height: 100%;
+  position: relative;
 }
 
 .navlink {
   composes: labelLarge from "../../styles/globals.css";
-  bottom: 20px;
   text-decoration: none;
   position: relative;
-  bottom: 0px;
+  padding-bottom: 6px;
+  border-bottom: 4px solid transparent;
```

Comment:
yuhhh i think right here is the reason for the bug with the navbar. Add a `cursor: pointer` here maybe?